src/backend/tests/unmocked/design-review.test.ts

```diff
@@ -0,0 +1,99 @@
+import { batmanAppAdmin, financeMember } from '../test-data/users.test-data';
+import DesignReviewsService from '../../src/services/design-reviews.services';
+import { AccessDeniedException } from '../../src/utils/errors.utils';
+import { createTestDesignReview, createTestUser, resetUsers } from '../test-utils';
+import prisma from '../../src/prisma/prisma';
+import { assert } from 'console';
+import { DesignReview, DesignReviewStatus } from 'shared';
+
+describe('Design Reviews', () => {
```

Comment:
These tests are taking on a lot more than they need to, meaning they are becoming very complicated. Take a look at our other unmocked tests and see the structure they fall into, making a call to the endpoint in a specific test case, and checking against what we expect in that case. Anything beyond that is really testing other endpoints or functionality, which is too much overhead for these tests

Reply:
Writing concise, effective tests can be tough to figure out, so let me (or a tech lead) know if you need help and the other examples aren't helping